source/common/redis/command_splitter_impl.cc

```diff
@@ -147,9 +147,12 @@ InstanceImpl::InstanceImpl(ConnPool::InstancePtr&& conn_pool, Stats::Scope& scop
       stats_{ALL_COMMAND_SPLITTER_STATS(POOL_COUNTER_PREFIX(scope, stat_prefix + "splitter."))} {
   // TODO(mattklein123) PERF: Make this a trie (like in header_map_impl).
   addHandler(scope, stat_prefix, "expire", all_to_one_handler_);
+  addHandler(scope, stat_prefix, "get", all_to_one_handler_);
```

Comment:
OK, thanks. This is a pretty trivial change so IMO might as well just do it now.